src/query/block/types.go

```
@@ -62,6 +78,19 @@ type SeriesIter interface {
 	Meta() Metadata
 }
 
+// UnconsolidatedSeriesIter iterates through a block horizontally
+type UnconsolidatedSeriesIter interface {
+	Iterator
+	// Current returns the current series for the block
+	Current() (UnconsolidatedSeries, error)
```

Current() typically panics rather than returning an error

---

all other Current() also return error